src/Dfe.PlanTech.Web/Program.cs

```diff
@@ -27,6 +28,8 @@
 }
 
 builder.Services.AddCustomTelemetry();
+builder.Services.Configure<ErrorMessages>(builder.Configuration.GetSection("ErrorMessages"));
+builder.Services.Configure<ContactOptions>(builder.Configuration.GetSection("ContactUs"));
```

Comment:
This is done a little differently to the other options like setup which uses records, and seems simpler, but I'm wondering how different end result is, any idea?
Not suggesting you should change this - just interested in which to use when

Reply:
Where are the setup options registered, I can take a look? I thought this was just a simple way to use automatic binding. Though they probably should be records: [54c9c1a](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/891/commits/54c9c1a4e677ace699020c9f0b43c97543224a8c)